packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -49,7 +49,7 @@ const esSearchStub = sandbox.stub();
 const esScrollStub = sandbox.stub();
 FakeEsClient.prototype.scroll = esScrollStub;
 FakeEsClient.prototype.search = esSearchStub;
-
+// is this es-client stuff still needed b/c granules and metrics?
```

Comment:
I think es-client is still needed. You can try to take it out.

Reply:
when I tried removing ES, I found that some of the ES vars were for Metrics, so it seemed like ES needs to stay for these tests